curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-83-path-sum-four-ways.english.md

```diff
@@ -7,21 +7,23 @@ forumTopicId: 302197
 
 ## Description
 <section id='description'>
-NOTE: This problem is a significantly more challenging version of Problem 81.
+
+**Note:** This problem is a significantly more challenging version of Problem 81.
+
 In the 5 by 5 matrix below, the minimal path sum from the top left to the bottom right, by moving left, right, up, and down, is indicated in bold red and is equal to 2297.
 
-$$
-\begin{pmatrix}
-\color{red}{131} & 673 & \color{red}{234} & \color{red}{103} & \color{red}{18}\\
-\color{red}{201} & \color{red}{96} & \color{red}{342} & 965 & \color{red}{150}\\
-630 & 803 & 746 & \color{red}{422} & \color{red}{111}\\
-537 & 699 & 497 & \color{red}{121} & 956\\
-805 & 732 & 524 & \color{red}{37} & \color{red}{331}
-\end{pmatrix}
-$$
-
-Find the minimal path sum, in matrix.txt (right click and
-"Save Link/Target As..."), a 31K text file containing a 80 by 80 matrix, from the top left to the bottom right by moving left, right, up, and down.
+<div style='text-align: center;'>
+  $\begin{pmatrix}
+  \color{red}{131} & 673 & \color{red}{234} & \color{red}{103} & \color{red}{18}\\\\
+  \color{red}{201} & \color{red}{96} & \color{red}{342} & 965 & \color{red}{150}\\\\
+  630 & 803 & 746 & \color{red}{422} & \color{red}{111}\\\\
+  537 & 699 & 497 & \color{red}{121} & 956\\\\
+  805 & 732 & 524 & \color{red}{37} & \color{red}{331}
+  \end{pmatrix}$
+</div>
+
+Find the minimal path sum from the top left to the bottom right by moving left, right, up, and down in `matrix`, a 2D array containing an 80 by 80 matrix.
```

Comment:
```suggestion
Find the minimal path sum from the top left to the bottom right by moving left, right, up, and down in `matrix`, a 2D array, defined in the background, containing an 80 by 80 matrix.
```
I'm not sure my change is necessary, I was just thinking that someone might wonder what `matrix` is.

Reply:
No, I agree. It's best to mention that it's defined in the background to prevent confusion. I'll go through and make sure to include this note with other things that are in the `Before Test` section.